examples/moog_filter.rs

```diff
@@ -0,0 +1,238 @@
+/*
```

Comment:
Additionally, we can use `//!` instead of the multiline comment here, which will nicely let cargo doc show this information in the API docs

Reply:
Not sure if i did this right, but I changed to those /// documentation comments

Reply:
Latest commit looks good! Thanks